odoo/addons/base/models/res_bank.py

```diff
@@ -84,6 +84,7 @@ def _get_supported_account_types(self):
     sequence = fields.Integer(default=10)
     currency_id = fields.Many2one('res.currency', string='Currency')
     company_id = fields.Many2one('res.company', 'Company', related='partner_id.company_id', store=True, readonly=True)
+    country_code = fields.Char(related='partner_id.country_code', string="Country Code")
```

Comment:
I would challenge the need to put this in base.
It's not a big issue, but as it's only used by accounting modules I would suggest having the field inside the account module itself. (It won't change much, but we may as well keep the base module as clean as possible)

Reply:
noted on this, the reason for is because `l10n_hk`, `l10n_sg`, `l10n_vn` etc. , res_bank_views.xml are all inheriting from 'base.view_partner_bank_form'. If we put it on account, then I would need to change the inherit_id for all these views that use country_code. It was only for convenience :grin: 
But noted, I'll move it to account

Reply:
Hmm maybe base is ok then... It's not better to go change inherits everywhere

Reply:
noted!